tsconfig.json

```diff
@@ -6,10 +6,10 @@
     "resolveJsonModule": true,
     "rootDir": ".",
     "useUnknownInCatchVariables": false,
-    "declaration": false,
     "lib": ["DOM"]
   },
   "include": [
+    "package.json",
```

Comment:
Why?

Reply:
The new `gts` tsconfig file, which this inherits from, sets `"composite": true`. This, for some reason makes it so JSON files also have to be added in the `include` list.

```
lib/index.ts:5:22 - error TS6307: File '/home/runner/work/embedded-host-node/embedded-host-node/package.json' is not listed within the file list of project '/home/runner/work/embedded-host-node/embedded-host-node/tsconfig.build.json'. Projects must list all files or use an 'include' pattern.
  The file is in the program because:
    Imported via '../package.json' from file '/home/runner/work/embedded-host-node/embedded-host-node/lib/index.ts'
    Imported via '../package.json' from file '/home/runner/work/embedded-host-node/embedded-host-node/tool/prepare-optional-release.ts'
    Imported via '../package.json' from file '/home/runner/work/embedded-host-node/embedded-host-node/tool/prepare-release.ts'
```

Related issue - https://github.com/microsoft/TypeScript/issues/25636

Reply:
Oh, I see, so this is specifically because we load this file from TypeScript.